server/src-rsr/migrate_from_11_to_12.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR REPLACE FUNCTION hdb_catalog.insert_event_log(schema_name text, table_name text, trigger_name text, trigger_id text, op text, row_data json) RETURNS void AS $$
```

Comment:
should we return affected rows?

Reply:
we should return event id actually so that the frontend can poll the response immediately.